eslint-plugin-prettier.js

```diff
@@ -164,7 +167,8 @@ module.exports = {
               : null;
 
             const prettierFileInfo = prettier.getFileInfo.sync(filepath, {
-              ignorePath: '.prettierignore'
+              ignorePath: '.prettierignore',
+              withNodeModules
```

Comment:
Rather than pass in a single value within the options here, I think it would be more future-compatible to pass in an object that is merged with some defaults. That means that if prettier decides to add additional options to getFileInfo() then we won't need to make additional changes to accommodate them.

Something like this (written but not tested):

```js
const fileInfoOptions = Object.assign(
    {},
    { ignorePath: '.prettierignore' }
    eslintFileInfoOptions
)

const prettierFileInfo = prettier.getFileInfo.sync(filepath, fileInfoOptions);
```

Where `eslintFileInfoOptions` is defined above as something like this: (written but not tested)

```js
const eslintFileInfoOptions = context.options[1] && context.options[1].fileInfoOptions || {}
```